tests/runtime.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+cd ./tests/python
+
+# read an argument to run a specific test file
+# no argument to run the entire test suite
+if [ -z "$1" ]; then
+    sudo python -m unittest discover --pattern=*.py
```

Comment:
I think it'd be best if the `sudo` was left out of this script and explicitly typed by the person running the tests